spec/request/folders_request_spec.rb

```diff
@@ -65,4 +65,34 @@
       end
     end
   end
+
+  describe '#destroy' do
+    context 'with valid folder id' do
+      let(:id) { 613_557 }
+
+      it 'response to DELETE #destroy' do
+        VCR.use_cassette("sm/folders/#{user_id}/delete_valid") do
+          delete "/v0/messaging/health/folders/#{id}"
+        end
+
+        expect(response).to be_success
+        expect(response).to have_http_status(:no_content)
+      end
+    end
+
+    context 'with non-existing id' do
```

Comment:
Adding it to my list of questions I'm about to send the MHV API team. I guess it's good you can't delete a system folder, but most of MHV's errors have been really accurate so far so just feels strange this one isn't.